app/steps/step_navigation.rb

```diff
@@ -13,6 +13,10 @@ class StepNavigation
       PersonalDetailController,
       HouseholdMembersOverviewController,
     ],
+    "Money & Income" => [
+      IncomeChangeController,
+      IncomeChangeExplanationController,
+    ],
```

Comment:
The number of steps in this app are growing like weeds! :seedling: 

Good weeds!